src/GPEmulator.jl

```diff
@@ -117,12 +117,12 @@ function GPObj(inputs, data, package::GPJL; GPkernel::Union{K, KPy, Nothing}=not
         GPkernel_i = deepcopy(GPkernel)
         # inputs: N_samples x N_parameters
         # data: N_samples x N_data
-        logstd_obs_noise = log(sqrt(0.5)) # log standard dev of obs noise
+        logstd_obs_noise = log(sqrt(0.01)) # log standard dev of obs noise
```

Comment:
Should we make this dependent on the ensemble covariance? I also think that Ollie wanted to base this on the SVD stuff.

Reply:
Yes, we should! This will come in another (probably the next) PR though - the purpose of this one is merely to extend the test coverage. The change of the hard-coded regularization noise from `log(sqrt(0.5))` to `log(sqrt(0.01))` is just there because I experimented with it after our discussion and thought I could as well leave it at a smaller value until we have implemented the SVD (since the idea is that the regularization noise should be smaller than the learned white noise, a criterion that is more likely to be met the smaller the regularization noise is - speaking of which, one of the next PRs should also address the issue of naming all these noises (white noise, observational noise, regularization noise etc.) in a consistent way that minimizes confusion as much as possible).

Reply:
Sounds good. And yes, I think with all the covariances involved, being precise in the notation would be helpful.